metricbeat/metricbeat.reference.yml

```diff
@@ -163,19 +163,18 @@ metricbeat.modules:
 
 #------------------------------- Docker Module -------------------------------
 - module: docker
-  metricsets:
-    - "container"
-    - "cpu"
-    - "diskio"
-    - "event"
-    - "healthcheck"
-    - "info"
-    #- "image"
-    - "memory"
-    - "network"
-  hosts: ["unix:///var/run/docker.sock"]
+  #metricsets:
```

Comment:
that's a fair point, we could separate the list beetween default metricsets and non default maybe? Something like:
```
  # Default metricsets:
  #  - container
  #  - cpu
  #  - diskio
  #  - event
  #  - healthcheck
  #  - info
  #  - memory
  #  - network
  # Disabled by default:
  #  - image
```

Reply:
Let's do something like this. I think the above is not 100% obvious yet and we could improve the Design by introducing newlines or similar. And the wording for consistency should probably be `enabled` by default as otherwise we need to explain the user what `default` means.

Reply:
I think it was better before, from the user perspective they only had to comment in/out specific lines to enable or disable specific modules.
With this proposal to disable one metricset they have to comment in the `metricsets:` line and all the default ones except the one they want to disable.

Reply:
I agree with Jaime, also users are already used to comment in/out in this config file to enable/disable metricsets.

Reply:
The above is almost what have right now in the "short" config I think? The reason it's like this and not the way we have it in the reference config is the defaults. Assuming have `cpu, diskio` as default metricsets in the code in 6.3 and in 6.4 we decided to also enabled `info` as default in the code, users which didn't change the config would automatically get the new default. If we have it commented out, users that upgrade and have not modified the config, will not get the new metricset automatically.

Reply:
@ruflin yes, you are right, better to support default use cases.